docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -552,18 +568,18 @@ link:https://datatracker.ietf.org/doc/html/rfc7636[Proof Key for Code Exchange]
 While PKCE is of primary importance to public OpenID Connect clients, such as SPA scripts running in a browser, it can also provide an extra level of protection to Quarkus OIDC `web-app` applications.
```

Comment:
```suggestion
While PKCE is of primary importance to public OpenID Connect clients, such as SPA scripts running in a browser, it can also provide extra protection to Quarkus OIDC `web-app` applications.
```